### PR TITLE
Authentication And Routing

### PR DESCRIPTION
**Pull request by SaurabhChawda**
Added Authentication and Routing in Video Library.
Screenshot (370)
Screenshot (369)
Screenshot (372)

### CODE REVIEW

File: src/Reducers/AuthReducer.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export function Reducer(state, action) {
+  switch (action.type) {
+    case "SIGNUP":
```
Comment (sameep-s) [inline src/Reducers/AuthReducer.jsx:3]: You can export these actions as constants as a good practice it will decrease number of typos.

**Review by sameep-s — APPROVED** [review]
You can export reducer actions as constants otherwise code looks good you can merge.

File: src/App.css [hunk 2]
```diff
@@ -16,24 +16,28 @@ a {
...
-[theme="light"] {
+[theme="dark"] {
   --primary-color: #ff8c44;
   --primary-bg-color: #151515;
   --secondary-bg-color: rgb(34, 31, 31);
   --primary-theme-color: white;
   --secondary-theme-color: black;
+  --white-color: white;
+  --black-color: black;
   --font-size-XX-large: clamp(3rem, 12vw, 10rem);
```
Comment (ankit9015) [inline src/App.css:27]: I like that you are using clamp function

**Review by ankit9015 — APPROVED** [review]
Code is well structured and good to merge

File: src/Reducers/AuthReducer.jsx (hunk 1 above)
Comment (SaurabhChawda) [reply src/Reducers/AuthReducer.jsx:3]: I will Update.

**Comment by SaurabhChawda** [comment]
> You can export reducer actions as constants otherwise code looks good you can merge.

ThankYou

File: src/App.css (hunk 2 above)
Comment (SaurabhChawda) [reply src/App.css:27]: ThankYou

**Pull request merged**